src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -150,6 +154,10 @@ struct Input
       ->description("Center for object (x,y[,z])")
       ->expected(2, 3);
 
+    app.add_flag("--random-spacing,!--no-random-spacing", randomSpacing)
```

Comment:
Disabling random spacing is useful because the check can eliminate false positives when it can predict the point spacing.